resources/public/api/conf/3.0/common/pathParameters.yaml

```diff
@@ -34,6 +34,21 @@ components:
       in: path
       description: |
         The type of business the summary calculation is for.
+        
+        Limited to the following possible values for tax years before 2025-26:
+
+        - foreign-property-fhl-eea
+        - foreign-property
+        - uk-property-non-fhl
```

Comment:
Please update to `uk-property`. `uk-property-non-fhl` has been updated to `uk-property`